_data/guides.yaml

```diff
@@ -0,0 +1,18 @@
+classname: guides
+header: Wild<strong>Fly</strong> Elytron<br>Developer Guides 
+description: >
+  Find the guide for your operating system to get your development environment setup,
+  and then check out our <a href="/wildfly-elytron/getting-started-for-developers"><strong>
+  getting started for developers guide</strong></a> to get familiar with the Elytron security project
+  and the WildFly application server.
+  
+elements:
+  - header: <a href="/wildfly-elytron/guides/setup-elytron-linux/"><strong>Linux & WSL</strong></a>
+    icon: /assets/images/guides/icon-linux.png
+    icon_link: /wildfly-elytron/guides/setup-elytron-linux/
+  - header: <a href="/wildfly-elytron/guides/setup-elytron-windows/"><strong>Windows 10/11</strong></a>
+    icon: /assets/images/guides/icon-windows.png
+    icon_link: /wildfly-elytron/guides/setup-elytron-windows/
+  - header: <a href="/wildfly-elytron/guides/setup-elytron-macos/"><strong>macOS</strong></a>
+    icon: /assets/images/guides/icon-macos.png
+    icon_link: /wildfly-elytron/guides/setup-elytron-macos/
```

Comment:
For the OS logos, I'm just wondering if it's better to avoid changing the official logo colours. What if we use the same images from here: https://code.visualstudio.com/download

Reply:
A minor detail - do you think it would be possible to reduce the space in between the logos by a bit so it's not as spread out? No worries if this isn't easy, we can leave it.